composer/utils/dist.py

```diff
@@ -256,6 +294,9 @@ def broadcast_object_list(object_list: List[Any], src: int = 0) -> None:
     Returns:
         None:  ``object_list`` will be modified in-place and set to values of ``object_list`` from the ``src`` rank.
     """
+    if is_torch_xla_installed():
+        raise RuntimeError(f'broadcast_object_list is not supported with torch_xla!')
```

Comment:
```suggestion
        raise RuntimeError(f'broadcast_object_list is not supported with torch_xla')
```
(very) minor nit, feel free to ignore. Prefer no `!` to be consistent